docs/release_flow.md

```diff
@@ -0,0 +1,65 @@
+# About Release FLow
+
+1. For general guidance about using GitHub actions, you can take a look at [Github Actions Guide](https://docs.github.com/en/actions/guides). 
+2. We have a total of 7 active release workflows. These are located under [Workflows](https://github.com/REAN-Foundation/reancare-service/tree/develop/.github/workflows).
+
+## Table Of Contents
+- [Branching Strategy](#Branching-Strategy)
+  - [Feature](#Feature-Branch)
+  - [Release](#Release-Branch)
+- [Develop to Main Branch Workflow](#Develop-to-Main-branch-Workflow)
+  - [Develop Branch Workflow](#Develop-Branch-Workflow)
+  - [Main branch Workflow](#Main-branch-Workflow)
+- [Release Workflows](#Workflow-Explained)
+  - [PR-ci-cd](#PR-ci-cd)
+  - [Dev-ci-cd](#Dev-ci-cd)
+  - [UAT-ci-cd](#UAT-ci-cd)
+  - [PROD-ci-cd](#PROD-ci-cd)
+  - [AHA-UAT-ci-cd](#AHA-UAT-ci-cd)
+  - [AHA-PROD-ci-cd](#AHA-PROD-ci-cd)
+
+
+
+## Branching Strategy
+
+We are using GitFlow Branching [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
+
+* main : The main branch serves as the stable and production-ready branch, where all the changes from release branches are merged and tested before deployment.
```

Comment:
Basic formatting rules! When you type a key and then describe it, the key should be highlighted by either making it bold or putting it as code block like `main`